fill-dirt.lic

```diff
@@ -0,0 +1,31 @@
+custom_require.call(%w[common common-items drinfomon])
```

Comment:
Do you plan to document this on elanthipedia?

Reply:
Sure, I can do that!

Reply:
Can you add a header documentation block to the script?

Reply:
Done!